src/transformers/configuration_utils.py

```diff
@@ -317,6 +317,9 @@ def from_pretrained(cls, pretrained_model_name_or_path: Union[str, os.PathLike],
             proxies (:obj:`Dict[str, str]`, `optional`):
                 A dictionary of proxy servers to use by protocol or endpoint, e.g., :obj:`{'http': 'foo.bar:3128',
                 'http://hostname': 'foo.bar:4012'}.` The proxies are used on each request.
+            use_auth_token (:obj:`str` or `bool`, `optional`):
+                Specify token to use as Bearer authorization for remote files. If true, will get token from
+                ~/.huggingface.
```

Comment:
```suggestion
                The token to use as bearer authorization for the remote files. If :obj:`True`, will get the token
                generated when running :obj:`transformers-cli login` (stored in :obj:`~/.huggingface`).
```
Think it's better to tell the user what to do to get that token in the right place than where the token is stored (unless I misunderstood where that token is coming from).

This might also warrant a note like:
```
            .. note::

                Passing a token is required when you want to use a private model.
```
if the default is to not use the token when a user is logged in.

Reply:
makes sense @sgugger. I'll wait for other potential comments and batch them tomorrow.

Re. the note, where would I add it? e.g. https://huggingface.co/transformers/philosophy.html#main-concepts ?

Thoughts on your side on the "should make it enabled by default" question?

Reply:
I was thinking in the doc string of `from_pretrained`, but it can also go there. The more the merrier since people don't read all the doc.

Reply:
As for "should make it enabled by default", I have no strong feeling either way. It makes sense to me to have some added code where something won't work out of the box for all users. It also makes sense to want logged-in users to be able to use it automatically. I think we can start with your approach and see users thoughts.